test/test_respondent_controller.py

```diff
@@ -55,6 +55,24 @@ def populate_with_respondent(self, tran, session, respondent=None):
         account_controller.register_user(respondent, tran)
         return self.respondent
 
+    @transactional
+    @with_db_session
+    def populate_with_active_respondent(self, tran, session, respondent=None):
```

Comment:
I don't know that a completely new function to add an active respondent is necessary just pass it in to the populate with respondent that's why it has the respondent.get('status'). This seems unnecessary when the same can be done with code already there.

Reply:
👍 Could be a partial instead.

Reply:
We have an active respondent but yeah we could.